frontend/src/components/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -4,7 +4,7 @@ import { getNameOf } from 'helpers/helpers';
 import { useAppForm } from 'hooks/hooks';
 import { userSignUp as userSignUpValidationSchema } from 'validation-schemas/validation-schemas';
 import { DEFAULT_SIGN_UP_PAYLOAD } from './common';
-import styles from '../../Auth.module.scss';
+import styles from '../../auth.module.scss';
```

Comment:
Take a look at this [comment](https://github.com/BinaryStudioAcademy/bsa-2022-guruhub/pull/15#discussion_r939577838) regarding import order.
Also, use `import styles from components/auth/auth.module.scss`, instead of `../..`